WcaOnRails/app/jobs/timed_application_job.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class TimedApplicationJob < ApplicationJob
```

Comment:
For all we know, there actually already is such a thing in Rails 5. Someone adventurous can try deleting the file and finding out.